### ISSUE TITLE
/opt/anaconda1anaconda2anaconda3 in openmpi package

### ISSUE THREAD

**Issue by asmeurer** [ContinuumIO/anaconda-issues#447]
Reported on gitter.im/conda/conda by @michaelaye

```
could it be that openmpi on the python=2 dist has a packaging problem? I can’t seem to run any of the executables without getting a weird /opt… path reference:
$ which mpirun
/Users/klay6683/miniconda3/envs/py27/bin/mpirun
(py27)
klay6683 at MacL2938 in ~/src/petsc (maint)
$ mpirun -h
--------------------------------------------------------------------------
Sorry!  You were supposed to get help about:
    orterun:usage
But I couldn't open the help file:
    /opt/anaconda1anaconda2anaconda3/share/openmpi/help-orterun.txt: No such file or directory.  Sorry!
--------------------------------------------------------------------------
(py27)
klay6683 at MacL2938 in ~/src/petsc (maint)
$ mpic++ -h
Cannot open configuration file /opt/anaconda1anaconda2anaconda3/share/openmpi/mpic++-wrapper-data.txt
Error parsing data file mpic++: Not found
(py27)
```

**Comment by lzkelley** [comment]
+1.  Getting the same error.

**Comment by rainwoodman** [comment]
Is there any progress on this?

**Comment by rainwoodman** [comment]
Another issue on this is at https://github.com/conda/conda/issues/2277

**Comment by nehaljwani** [comment]
This has been fixed in `openmpi=1.6.3=1`
```bash
(root) inferno:~ nwani$ conda list openmpi
# packages in environment at /Users/nwani/m3:
#
openmpi                   1.6.3                         1
(root) inferno:~ nwani$ mpirun -h
mpirun (Open MPI) 1.6.3

Usage: mpirun [OPTION]...  [PROGRAM]...
Start the given program using Open RTE

...
Report bugs to http://www.open-mpi.org/community/help/
```
